drivers/ethernet/Kconfig.litex

```
@@ -0,0 +1,7 @@
+# Copyright (c) 2019 Antmicro
+# SPDX-License-Identifier: Apache-2.0
+
+config ETH_LITEX
```

```suggestion
config ETH_LITEX_LITEETH
```
Let's keep the naming consistent.

---

sure